common/pulseaudio-ctl.in

```diff
@@ -136,15 +136,15 @@ refreshcurvol() {
   # CURVOL=$(pacmd list-sinks|grep -A 15 '* index'| awk '/base volume: /{ print $5 }'|sed 's/%//')
   [[ $PCV -eq 0 ]] && 
   CURVOL=$(pacmd list-sinks|grep -A 15 '* index'| awk '/volume: /{ print $3 }' | grep -m 1 % |sed 's/[%|,]//g') ||
-    CURVOL=$(pacmd list-sinks|grep -A 15 '* index'| awk '/volume: front/{ print $5 }' | sed 's/[%|,]//g')
+    CURVOL=$(pacmd list-sinks|grep -A 15 '* index'| awk '/volume: [front/mono]/{ print $5 }' | sed 's/[%|,]//g')
```

Comment:
```suggestion
    CURVOL=$(pacmd list-sinks|grep -A 15 '* index'| awk '/volume: (front|mono)/{ print $5 }' | sed 's/[%|,]//g')
```

This is to match not a character in the set `{f, r, o, n, t, /, m}` but to match word `front` or `mono`.